demo/app.jsx

```diff
@@ -173,11 +174,11 @@ class App extends React.Component {
       {device.Name} - IsOpen: <span className={device.IsOpen ? 'open' : 'closed'}>
         {device.IsOpen ? 'true' : 'false'}
       </span> - <a href="#" onClick={(e) => this.handleOpen(e, device.Name)}>
-          open
```

Comment:
@matteosuppo pleeease, don't modify the formatting :smile:

Reply:
Sorry vscode decided it was better that way. I didn't even notice. Should I put it the way it was?

Reply:
I know, vscode shows its own will sometimes. Yes please :+1: